liquid-interpreter/src/context.rs

```diff
@@ -328,6 +348,11 @@ impl<'g> Context<'g> {
         CycleState { context: self }
     }
 
+    /// Access the block's `IfChangedState`.
+    pub fn ifchanged(&mut self) -> &mut IfChangedState {
```

Comment:
I need to look into a `registers` thing like ruby liquid to avoid having to add tag-specific knowledge to the `Context`.